spec/features/services/service_with_draft_spec.rb

```diff
@@ -80,7 +80,7 @@
       end
     end
 
-    context 'when the services provider is not in the users available providers' do
```

Comment:
if you don't need to keep the `js: true` for the test to run and pass properly, then it should be removed.

Reply:
I have removed the "js: true" in the latest commit.